src/components/EventList.js

```diff
@@ -132,6 +136,7 @@ class EventList extends Component<Props, State> {
           addSavedEvent={addSavedEvent}
           removeSavedEvent={removeSavedEvent}
           onPress={onPress}
+          testID={`event-card-${index}-${section.index}`}
```

Comment:
I don't love making the `testID` *less* unique by using the two indices (rather than the event ID). However, for the e2e tests, unlike the component-level tests, we can't load test data (and therefore know what the event ID will be), the data comes from Contentful itself. So given it's only a `testID` (and not something important like a `key`) I thought it might be ok to use the indices in this manner... Thoughts?

The reason each event card needs a `testID` is so the e2e tests can explicitly pick out the first event and test saving the event as well as tapping through to Event Details. Detox does provide an `.atIndex(n)` that I wanted to use, but there appears to be a bug because `.atIndex(0)` keeps picking random events in the middle of the list, off screen, so I can't test what's needed because the element won't be interactable